src/main/java/com/laserfiche/api/client/model/AccessKey.java

```diff
@@ -36,7 +38,7 @@ public static AccessKey createFromBase64EncodedAccessKey(String base64EncodedAcc
         try {
             accessKey = mapper.readValue(accessKeyStr, AccessKey.class);
         } catch (IOException e) {
-            throw new RuntimeException(e);
+            throw new RuntimeException(String.format("Invalid access key.%nDetail:%s", e));
```

Comment:
What we pass to an Exception class's argument is the exception message, and AFAIK the message string is usually a one-line string. so, probably it's better to replace the %n with a whitespace. Also adding a whitespace after : seems better.

Reply:
Fixed, thanks!